### ISSUE TITLE
tick_sched_timer and fwupd are using a lot of energy 

### ISSUE THREAD

**Issue by takov** [clearlinux/distribution#1924]
When switching to battery it gets drained fast. I don't have the time metrics but when I run powertop it  shows that "tick_sched_timer" uses a lot of energy (comparing to the other processes) ~4 W. 

See the image below:
![Batterry2](https://user-images.githubusercontent.com/3611602/80071064-ad510c00-854c-11ea-916e-1c94907722ea.png)

The using is constant and its between 4W and 2W
Also fwupd is using ~1W but on intervals. Some times it uses mW and some time >1W. Even saw it > 3W. But mostly it is ~1W

Laptop: Dell 5520
CPU: Intel® Core™ i7-7820HQ CPU @ 2.90GHz × 8 
Graphics: Mesa Intel® HD Graphics 630 (KBL GT2)
RAM: 15.5 GiB

Distribution:  Clear Linux OS
Installed version: 32890

**Comment by fenrus75** [comment]
hmm fwupd is not supposed to be started by default
that's a bug

**Comment by ahkok** [comment]
`fwupd` is an optional component, I assume this person has explicitly installed it?

**Comment by takov** [comment]
No, I haven't. Most probably it came from "desktop" bunlde but it is "clr-fwupd-hooks". Don't know if it is the full "fwupd" package. 

See the image below: 
![image](https://user-images.githubusercontent.com/3611602/80182167-f4073a80-860e-11ea-9060-18f0eb498ccd.png)

And the bunlde list:
[bundle_list.txt](https://github.com/clearlinux/distribution/files/4527080/bundle_list.txt)

I am using flatpak and "desktop" is in the prerequisites for installing flatpaks. 

Hope this help.

**Comment by ahkok** [comment]
I'm severing that link right now. We've decided a while back that we wanted to make `fwupd` an optional and non-present-by-default component, and this shouldn't have been installed by default.
